class.jetpack-client-server.php

```diff
@@ -226,7 +227,7 @@ function get_token( $data ) {
 				'Accept' => 'application/json',
 			),
 		);
-		$response = Client::_wp_remote_request( Jetpack::fix_url_for_bad_hosts( Jetpack::connection()->api_url( 'token' ) ), $args );
+		$response = Client::_wp_remote_request( Manager::fix_url_for_bad_hosts( Jetpack::connection()->api_url( 'token' ) ), $args );
```

Comment:
this looks weird since we are calling the manager, once static and the second time as an instance. should we maike `api_url` call `fix_url_for_bad_hosts` instead?

Reply:
The results of api_url() aren’t passed to fix_url… in every context. For that reason, I don’t think we should make api_url() automatically invoke/decorate its return value with fix_url… .
<img width="828" alt="Screen Shot 2019-09-06 at 6 07 08 PM" src="https://user-images.githubusercontent.com/3246867/64463262-5e478b00-d0d1-11e9-8a69-ef4abea3793a.png">

Reply:
The `fix_url_for_bad_hosts` method is now in the new Utils class. See a142b9c.